css/converse.css

```diff
@@ -1160,7 +1160,7 @@ readers do not read off random characters that represent icons */
   content: "\f52b"; }
 
 .fa-dot-circle:before {
-  content: "\f192"; }
+  content: "\f111"; }
 
```

Comment:
Did you modify this file directly? I don't see where you made this change in any scss (Sass) file.

In any case, this is not the right approach. These font-awesome classes should be considered as constants, they never change.

Instead, if you want to show a solid circle, then you need to use a different class, not modify `.fa-dot-circle`.

Looks like the classes you need to use is `fas fa-circle`, that's a solid circle.
See here: https://fontawesome.com/icons/circle?style=solid

Reply:
Ah this should have been obvious, I'll fix this up.

Reply:
I will need to add a new class name so that the away icon does not override the online icon, since both use the fa-circle class name. Do you have a system or preference for amending the class name for the away fa-circle?
For example, I notice the 'Away for long' status uses fa-circle, but it has the preceding class 'far' rather than 'fa'.

Reply:
We've been using the fontawesome class names to style the icons, but this isn't really a good idea, we need our own class names, as you've figured out.

So please add the class `chat-status` to all 4 status icons, and then another class `chat-status--${show}` as well.

So for the away icon the classes are `fa fa-circle chat-status chat-status--away` and for the online icon it will be `fa fa-circle chat-status chat-status--online`.